test/useFormState.test.js

```diff
@@ -232,6 +254,30 @@ describe('onChange updates inputs value', () => {
     click();
     expect(changeHandler).toHaveBeenLastCalledWithValues({ radio: 'option' });
   });
+
+  it('updates value for type "select"', () => {
+    const values = ['option_1', 'option_2'];
+    const { change, changeHandler } = renderSelect('select', 'select', values);
+    expect(changeHandler).toHaveBeenLastCalledWithValues({ select: '' });
+    change({ value: 'option_1' });
+    expect(changeHandler).toHaveBeenLastCalledWithValues({
+      select: 'option_1',
+    });
+  });
+
+  it('updates value for type "selectMultiple"', () => {
+    const values = ['option_1', 'option_2', 'option_3'];
+    const { change, changeHandler } = renderSelect(
+      'selectMultiple',
+      'select',
+      values,
+    );
+    expect(changeHandler).toHaveBeenLastCalledWithValues({ select: [] });
+    change({ value: 'option_1' });
```

Comment:
> I'm currently struggling for testing multiple select, because it's a CTRL + click event and the target value is only the last selected option.

I wouldn't stress about selecting multiple this way because it's a browser behavior and we shouldn't be testing it in this library. I think what you have is good! 

That said, it'd be nice if the tests can verify that the library actually updates the form state when we select _multiple elements_ or when we _deselect_ an element.

One way to do that is to manipulate the underlying DOM options via the `select` element itself, before we fire the `change` event:

```js
it('updates value for type "selectMultiple"', () => {
  const values = ['option_1', 'option_2', 'option_3'];
  const { select, change, changeHandler } = renderSelect(
    'selectMultiple',
    'select',
    values,
  );

  expect(changeHandler).toHaveBeenLastCalledWithValues({ select: [] });

  select.options[0].selected = true; // selecting one options
  change();
  expect(changeHandler).toHaveBeenLastCalledWithValues({
    select: ['option_1'],
  });

  select.options[1].selected = true; // selecting another option
  change();
  expect(changeHandler).toHaveBeenLastCalledWithValues({
    select: ['option_1', 'option_2'],
  });

  select.options[0].selected = false; // deselecting an option
  change();
  expect(changeHandler).toHaveBeenLastCalledWithValues({
    select: ['option_2'],
  });
});
```